### ISSUE TITLE
About training settings.

### ISSUE THREAD

**Issue by ZitSaibok** [zju-vipa/MEAT-TIL#1]
Before the code is uploaded, may I have some detailed information about the training settings?
Such as data augmentation, learning rate scheduler, warmup epochs, weight decay, etc.

**Comment by xmq1221** [comment]
Please tell me your concern or email me, thanks for the comment.
